dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -534,20 +532,19 @@ public GHRepository getRepository(String repositoryId) {
     // The documentation doesn't list the possibilities https://github-api.kohsuke.org/apidocs/org/kohsuke/github/GHRef.GHObject.html#getType(),
     // but I'll assume it mirrors the 4 Git types: blobs, trees, commits, and tags.
     private String getCommitSHA(GHRef ref, GHRepository repository, String refName) throws IOException {
-        String sha = null;
+        String sha = ref.getObject().getSha();
         if ("commit".equals(ref.getObject().getType())) {
             sha = ref.getObject().getSha();
         } else if (ref.getObject().getType().equals("tag")) {
-            GHTagObject tagObject = repository.getTagObject(sha);
-            sha = tagObject.getObject().getSha();
+            sha = repository.getTagObject(ref.getObject().getSha()).getObject().getSha();
```

Comment:
You could do `repository.getTagObject(sha)` instead of `repository.getTagObject(ref.getObject().getSha())` since `sha` already has that value